requirements.txt

```diff
@@ -1,6 +1,6 @@
 #
-# This file is autogenerated by pip-compile with python 3.10
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.10
+# by the following command:
```

Comment:
I was a bit surprised that I didn’t see a corresponding change to `requirements.in`.
And then I checked that the `requirements.in` does not include any version ranges at all. Would it make sense to add those?

Reply:
i guess it's just a good way of making sure we're using the most up-to-date version of the packages, it's always easier to run pip-compile and test everything periodically